.agents/skills/ai-code-review/scripts/fetch_ai_pr_comments.sh

```
@@ -0,0 +1,225 @@
+                comments(first: 50) {
+                  nodes {
+                    databaseId
+                    url
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    ' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)